src/map/clif.c

```diff
@@ -7106,7 +7106,7 @@ void clif_vendinglist(struct map_session_data* sd, int id, struct s_vending* ven
 
 #if PACKETVER < 20150226
 	const int item_length = 22;
-#elif PACKETVER < 20161228
+#elif PACKETVER < 20160921
```

Comment:
@Lemongrass3110  already added packet length 59for PACKETVER condition 20160921 in the void clif_vendinglist..